### PR TITLE
Add support for entrypoint in stack files

### LINKED ISSUE #2949: Stack yml does not support entrypoint
Entrypoint is supported by api.

### PR DESCRIPTION
**Pull request by kke**
Fixes #2949

Adds / fixes support for the `entrypoint` keyword in stack/compose YAMLs.

### CODE REVIEW

**Comment by kke** [comment]
For reference, https://github.com/kontena/kontena/blob/9defedc8bc82cea6eb48ee2708bd6a69e2424829/cli/lib/kontena/cli/stacks/service_generator.rb includes all the keys imported from compose yml's.

Should cross check with https://kontena.io/docs/references/stack-file-vs-compose.html

**Comment by kke** [comment]
I think `depends_on`, `logging` and `env_file` are not supported when extending.

**Comment by SpComb** [comment]
I think this should have an e2e test.

**Comment by SpComb** [comment]
I don't mean a test spec that runs `kontena stack validate`, I mean an actual e2e spec that deploys the stack and looks at the deployed container entrypoint.

That kind of real e2e spec is needed to make sure that e.g. the CLI is sending the right form of `{ "entrypoint": ...}`  JSON that doesn't just get ignored/rejected by the server, or possibly even cause the container create to fail on the agent (is it an array or a string in the Docker API? Both, I think...). The current specs don't cover that.

**Comment by SpComb** [comment]
Tested to work with both the `/w/w` entrypoint and in `network_mode: host`, I think I'll write up an e2e spec for this next.

File: test/spec/support/container_helper.rb [hunk 1]
```diff
@@ -24,4 +26,10 @@ def container_id(name)
     k.out.match("^(.+\/#{name})")[1]
   end
 
-end
\ No newline at end of file
+  # @raise [RuntimeError]
+  # @return [Hash]
+  def inspect_container(container_id)
```
Comment (SpComb) [inline test/spec/support/container_helper.rb]: This e2e spec helper is identical to what's also in #2951

**Pull request merged**